src/axom/core/memory_management.hpp

```diff
@@ -34,50 +41,40 @@ constexpr int INVALID_ALLOCATOR_ID = -1;
  * \param [in] resource_type the Umpire resource type
  * \return ID the id of the predefined umpire allocator.
  */
-inline int getResourceAllocatorID(
+inline int getUmpireResourceAllocatorID(
   umpire::resource::MemoryResourceType resource_type )
 {
   umpire::ResourceManager& rm = umpire::ResourceManager::getInstance();
   umpire::Allocator alloc     = rm.getAllocator( resource_type );
   return alloc.getId();
 }
 
-/*!
- * \brief Returns the umpire allocator associated with the given ID.
- * \param [in] allocatorID the ID of the allocator to get.
- */
-inline umpire::Allocator getAllocator( int allocatorID )
-{
-  return umpire::ResourceManager::getInstance().getAllocator( allocatorID );
-}
-
-/*!
- * \brief Sets the default memory space to use. Default is set to HOST
- * \param [in] allocator the umpire::Allocator to make default.
- */
-inline void setDefaultAllocator( umpire::Allocator allocator )
-{
-  umpire::ResourceManager::getInstance().setDefaultAllocator( allocator );
-}
+#endif
```

Comment:
I think it might still be useful to keep this ``getAllocator(int) -> umpire::Allocator`` function in the ``ifdef AXOM_USE_UMPIRE`` section.

If we are removing it,  it might be a good idea to add a note to the doxygen to let users know how they can convert an int or string to an ``Allocator``.

Reply:
They can just get that by calling:

```
umpire::ResourceManager::getInstance().getAllocator(int)
```

Don't see a real need to provide a wrapper around that.

Reply:
As far as I see, the value is in documenting to our users that they can convert the ``int`` to an ``umpire::Allocator``.

If we don't have the function, it would be helpful to note the above in the ``doxygen``.

Reply:
Added a comment.